cql2pgjson/src/test/resources/templates/db_scripts/foreignKey.json

```diff
@@ -1,7 +1,20 @@
 {
   "tables": [
     {
-      "tableName": "tablea"
+      "tableName": "tablea",
+      "index": [
+        {
+          "fieldName": "prefix",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": false
+        }
+      ], "foreignKeys": [
+        {
+          "fieldName": "tabledId",
+          "targetTable": "tabled"
+        }
+      ]
```

Comment:
Can you avoid having a foreign key in tablea and instead use the existing foreign key from tableb to tablea? Reusing the same foreign key relation makes it more easy to follow the child->parent and parent->child tests.